docs/farming.md

```diff
@@ -19,17 +20,46 @@ You can find these executables in the [Releases](https://github.com/subspace/sub
 
 > This is a ***non-incentivized*** testnet. Meaning there are no rewards in place at this time, and has absolutely no financial benefit to being run at this time.
 
-### A. To Farm With The Network (Online)
+<details><summary> 🖼️ Windows Instructions (Click to Expand)</summary>
+<p>
 
-> NOTE: replace these: `subspace-node-x86_64-*-snapshot`, `subspace-farmer-x86_64-*-snapshot`
-> in the below commands with the name of the file you downloaded for your operating system.
+### 📝 Windows Installation
 
-**Mac/Linux**
+1. Download the executables for your operating system from the [Releases](https://github.com/subspace/subspace/releases) tab.
+2. open a terminal, on Windows this is `Command Prompt` or `Powershell`
```

Comment:
```suggestion
2. Open a terminal, on Windows this is `Command Prompt` or `Powershell`
```

Reply:
Also just out of curiosity, if PowerShell supports new lines in commands with sane syntax, I think we can safely require PowerShell specifically and not mention Command Prompt at all (it is legacy and those who really need to use it for whatever reason will have to figure out to put command into single like).

Reply:
> Also just out of curiosity, if PowerShell supports new lines in commands with sane syntax, I think we can safely require PowerShell specifically and not mention Command Prompt at all (it is legacy and those who really need to use it for whatever reason will have to figure out to put command into single like).

I agree, well just list Powershell.

as for the same syntax as the other commands Powershells equivalent to \ is a ` so we could semi-match the syntax and it would still be readable. 

```
./subspace-node-x86_64-*-snapshot `
--chain testnet `
--wasm-execution compiled `
--execution wasm `
--bootnodes "/dns/farm-rpc.subspace.network/tcp/30333/p2p/12D3KooWPjMZuSYj35ehced2MTJFf95upwpHKgKUrFRfHwohzJXr" `
--rpc-cors all `
--rpc-methods unsafe `
--ws-external `
--validator `
--telemetry-url "wss://telemetry.polkadot.io/submit/ 1" `
--name INSERT_YOUR_ID
```